### ISSUE TITLE
Support for issuing HS256 tokens?

### ISSUE THREAD

**Issue by SahAssar** [greenpau/caddy-auth-portal#59]
Hello!

Thanks for the awesome plugin! I have one question: I'm trying to use this in front of postgrest and postgrest seems to expect HS256 tokens, not HS512. Would it be possible to add support for using HS256 too? Or do you have any better idea how to make caddy-auth and postgrest work together? The docs for postgrest say this about it's `jwt-secret` config: "If it is set to a simple string value then PostgREST interprets it as an HMAC-SHA256 passphrase", but the only other option seems to be to use asymetric crypto.

Thanks!

**Comment by greenpau** [comment]
> Thanks for the awesome plugin!

@SahAssar , thank you! I have been putting a lot of work into this plugin. Would appreciate a star and a mention on the social media platforms you use. It is hard to promote a project, so I can use your help there :-)

> Would it be possible to add support for using HS256 too? 

@SahAssar , let's treat this as a feature. Will post Caddyfile directive to accomplish the thing you need in a moment.

**Comment by SahAssar** [comment]
Awesome, thanks! :+1:  I've stared and my plan is to write a blogpost about this setup when I have it setup and working. I think the combination of caddy, your jwt and auth-portal plugins and postgrest seem like a great way to easily kickstart many types of web backends.

**Comment by greenpau** [comment]
@SahAssar , you will be able specifying the signing method in this way:

```
      jwt {
        ...
        token_sign_method HS256
      }
```

**Comment by greenpau** [comment]
@SahAssar , the feature is available in [v1.2.13](https://github.com/greenpau/caddy-auth-portal/releases/tag/v1.2.13).

Please see https://github.com/greenpau/caddy-auth-portal#jwt-signing-method

**Comment by SahAssar** [comment]
Awesome, it works great! Thank you so much, I'll make sure to promote this as much as I can, both because it is a great plugin and also you really went above and beyond implementing a feature so quickly so that's the least I can do!

Please let me know if there is anything else I can do to help out in return :)
